.github/workflows/accept.yml

```diff
@@ -0,0 +1,186 @@
+      # use npm workaround specifying a package name to bypass install script in package.json
+      - name: NPM Install dependencies
+        run: docker exec ${{ steps.create.outputs.containerId }} npm install linux-os-info --unsafe-perm
+
+
+      # runs: node setup-liboboe.js && node-pre-gyp install --build-from-source
+      - name: NPM Install with Rebuild from source
+        run: docker exec ${{ steps.create.outputs.containerId }} npm run rebuild
+ 
+
+       # node-pre-gyp direct rebuild
+      - name: Build
+        run: docker exec ${{ steps.create.outputs.containerId }} npx node-pre-gyp rebuild
+
+      # artifacts are at:build/stage/nodejs/bindings/
+      - name: Package
+        run: docker exec ${{ steps.create.outputs.containerId }} npx node-pre-gyp package # requires clean rebuild
+
+      # *** IMPORTANT: 
+      # always include --s3_host flag regardless of node-pre-gyp defaults.
+      # node-pre-gyp can't publish over existing version, thus always attempt unpublished of current in staging bucket
+      - name: Clear Staging for version
+        run: docker exec ${{ steps.create.outputs.containerId }} npx node-pre-gyp unpublish --s3_host=staging
+
+      - name: Publish Package to Staging
+        run: docker exec ${{ steps.create.outputs.containerId }} npx node-pre-gyp publish --s3_host=staging
+
+      - name: Stop & Remove container
+        if: always()
+        run: |
+          docker stop ${{ steps.create.outputs.containerId }}
+          docker rm --force ${{ steps.create.outputs.containerId }}
+
+  load-prebuilt-group:
+    runs-on: ubuntu-latest
+    needs: build-group-publish
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+    - name: Checkout ${{ github.ref }}
+      uses: actions/checkout@v2
+
+      # *** comments for: prebuilt-group.json
+      # contains images that are unable to build "out of the box" and will install prebuilt
+    - name: Load target group data
+      id: set-matrix
+      run: .github/scripts/matrix-from-json.sh .github/config/prebuilt-group.json
+
+  prebuilt-group-install:
+    runs-on: ubuntu-latest 
+    needs: load-prebuilt-group
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.load-prebuilt-group.outputs.matrix)}}
+
+    env:
+      AO_TOKEN_PROD: ${{ secrets.AO_TOKEN_PROD }}
+
+    steps:
+      - name: Checkout ${{ github.ref }}
+        uses: actions/checkout@v2
+
+      - name: Create Container ${{ matrix.image }}
+        id: create
+        run: .github/scripts/container-from-matrix-input.sh ${{ matrix.image }}
+
+      # *** IMPORTANT: by default our package is installed from production bucket as defined in package.json. 
+      # to test current build pass --s3_host="staging"
+      - name: NPM Install Staging
+        run: docker exec ${{ steps.create.outputs.containerId }} npm install --unsafe-perm --s3_host=staging
```

Comment:
Similar to the fallback scenario comment, can the output from the npm install command or lack of compilation artifact be checked, to help verify that "install from pre-gypped" actually happened?

Reply:
See above.